### PR TITLE
[Comparison Tool] VET TEC Profile Page Updates

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Update the VET TEC version of the GI Bill Comparison Tool profile page.

## Testing done
Tested locally and QA tested

## Screenshots
image

## Acceptance criteria
- [x] The "Header" section displays the following information:
  - a. Provider Name (Populated using the WEAMS data.)
  - b. Provider City, State (Populated using the WEAMS data.)
  - c. Additional Resources Section:
    - i. "Learn more about VET TEC" link opens the following URL in a new window: https://www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/
    - ii. "Apply for VET TEC" link opens the following URL in a new window: https://www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/apply-for-vettec-form-22-0994/introduction
    - iii. "Submit a complaint through our Feedback System" link opens the following URL in a new window: https://www.va.gov/education/submit-school-feedback/introduction

- [x] The "Approved programs" section displays the following information:
  - a. The text: "Program data will soon be available for this provider. Learn more about VET TEC programs."
  - b. "Learn more about VET TEC programs" opens the following URL in a new window: https://www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/vettec-high-tech-program/

- [x] The "Contact Details" section displays the following information:
  - a. VET TEC Provider Physical Address (Populated using the "physical address/physical institution" fields from WEAMS)
  - b. VET TEC Provider Mailing Address (Populated using the "address/institution" fields from WEAMS)

- [x] The "Application process" section displays the following information:
  - a. Copy located in Content Documentation

- [x] The "Additional information" section displays the following information:
  - a. "Institution codes" header
  - b. "VA facility code:" (Populated using the Facility Code in the WEAMS data.)

- [x] Each section on the profile page is collapsible.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Comment by dmccurdybah** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

File: src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx [hunk 1]
```diff
@@ -1,15 +1,53 @@
 import React from 'react';
 
 const VetTecApprovedPrograms = () => (
```
Comment (U-DON) [inline src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx]: Should this be `VetTecApplicationProcess`?

**Review by U-DON — APPROVED** [review]

File: src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx (hunk 1 above)
Comment (dmccurdybah) [reply src/applications/gi/components/vet-tec/VetTecApplicationProcess.jsx]: Yes, thanks!

**Pull request merged**